src/main/groovy/com/benjaminsproule/swagger/gradleplugin/model/InfoExtension.groovy

```diff
@@ -0,0 +1,96 @@
+package com.benjaminsproule.swagger.gradleplugin.model
+
+import io.swagger.models.Info
+import org.gradle.api.Project
+
+class InfoExtension implements ModelValidator, Swagerable<Info> {
+    String description
+    String termsOfService
+    String title
+    String version
+    ContactExtension contact
+    LicenseExtension license
+    Project project
+
+    protected InfoExtension() {
```

Comment:
I'd like to think there is a nicer way to do this?

Reply:
The best solution is probably for me to write the tests using  [spock](http://spockframework.org/spock/docs/1.1/spock_primer.html) which has mocks and spies that are easier to use than the otb stuff for groovy.